### DISCUSSION TITLE
Clarification around Helm Install?

### DISCUSSION THREAD

**Discussion by evandam** [actions/actions-runner-controller#3072] [Questions]
Hi folks,

I have previously been managing installing Actions Runner Controller with the `actions-runner-controller` Helm chart located in https://actions-runner-controller.github.io/actions-runner-controller, but noticed at some point the [install docs](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller#installing-actions-runner-controller) changed to point to an OCI registry.

I was wondering if there's a typo in the docs? It's pointing to `oci://ghcr.io/actions/actions-runner-controller-charts/gha-runner-scale-set-controller`, which I'm assuming comes from https://github.com/actions/actions-runner-controller/tree/master/charts/gha-runner-scale-set-controller.

There's no `README.md` or documentation for the `gha-runner-scale-set-controller` chart, but there is for https://github.com/actions/actions-runner-controller/tree/master/charts/actions-runner-controller - which seems a bit more "public-facing" and like the one I would expect to use.

Can someone help point me in the right direction? Thanks in advance!

**Comment by nikola-jokic** [comment]
There are two modes in which you can install ARC now
1. Legacy mode: contains resources under `actions.summerwind.net` group. Maintained by the community and charts should be located in the same place as before.
2. Scale-set mode: contains resources under `actions.github.com` group. Maintained by GitHub and charts are packaged as oci images

**Reply by evandam** [reply]
I see, thanks for the response @nikola-jokic! I'll be moving over to the official charts and start using `ghcr.io/actions/actions-runner` for our runner images 👌 

Maybe just a small thing but do you think it would be reasonable to add a `README.md` to the `scale-set-controller` and `scale-set` directories in https://github.com/actions/actions-runner-controller/tree/master/charts just so folks browsing the repo know where to look?

Thanks again :raised_hands:
